Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/zipline .
+# 
+# To run the container:
+#
+#    docker run -v=/path/to/your/notebooks:/projects -p 8888:8888/tcp --name zipline -it quantopian/zipline
```

Comment:
The kicker here is that if I restart top the container and then restart it with `docker start zipline`, I can create notebooks fine!